### ISSUE TITLE
Loading without index.html

### ISSUE THREAD

**Issue by rwwagner90** [maxogden/menubar#255]
Is it possible to not load any html? I was to replace the app content on ready with `mb.window.loadURL();` and I do not want it to load any html files before that.

**Comment by amaurymartiny** [comment]
The default behavior is indeed showing a popup (a BrowserWindow) with some HTML inside when one clicks on the menubar icon. What should happen when you click on that icon in your case?

The [native-menu](https://github.com/maxogden/menubar/tree/master/examples/native-menu) is an example app where there's no index.html.

**Comment by rwwagner90** [comment]
@amaurymartiny we need to load a custom thing that is not just a static html page. We do this:

```js
  const emberAppLocation = 'serve://dist';

  // Load the ember application using our custom protocol/scheme
  mb.window.loadURL(emberAppLocation);
```

Right now, I put a path to an empty html file just to force no errors. If I could have it use the `serve://dist` path, I suppose I could skip this.

**Comment by amaurymartiny** [comment]
Does `const mb = menubar({ index: 'serve://dist' })` work?

The link PR actually updated docs to say that `index` option can take any URL.

**Comment by rwwagner90** [comment]
@amaurymartiny I think I tried that, and it didn't work. I think the problem is it tries to load it immediately and we have to use the app from `mb` to pass to the protocol serve stuff.

```js
const mb = menubar({
  dir: join(__dirname || resolve(dirname('')), '..', 'loading'),
  browserWindow: {
    height: 600,
    resizable: false,
    width: 320,
    webPreferences: {
      contextIsolation: false,
      preload: join(__dirname, 'preload.js'),
      nodeIntegration: true
    }
  },
  icon: join(__dirname || resolve(dirname('')), '..', 'resources/menubar-icons/icon.png'),
  preloadWindow: true
});

// Registering a protocol & schema to serve our Ember application
if (typeof protocol.registerSchemesAsPrivileged === 'function') {
  // Available in Electron >= 5
  protocol.registerSchemesAsPrivileged([{
    scheme: 'serve',
    privileges: {
      secure: true,
      standard: true
    }
  }]);
} else {
  // For compatibility with Electron < 5
  protocol.registerStandardSchemes(['serve'], { secure: true });
}
protocolServe({
  cwd: join(__dirname || resolve(dirname('')), '..', 'ember-dist'),
  app: mb.app,
  protocol
});
```

It's a chicken and the egg problem, I suppose.

**Comment by amaurymartiny** [comment]
@rwwagner90 Does https://github.com/maxogden/menubar/pull/257 solve your issue?

**Comment by rwwagner90** [comment]
@amaurymartiny I am not sure. I tried to use `"menubar": "maxogden/menubar#6028e52f79e215ec010353353616c24faf9857e7",` and I get an error trying to run it `Error: Cannot find module '/Users/rwwagner/shipshape/tinct/electron-app/node_modules/menubar/lib/index.js'. Please verify that the package.json has a valid "main" entry` I guess maybe it needs to be built or released and I can't try it this way?

**Comment by rwwagner90** [comment]
@amaurymartiny I did a `yarn build` in the menubar folder and fixed that, and it seems to work great! Definitely excited to get this in and thanks for the super quick turnaround time! 😃

**Comment by amaurymartiny** [comment]
Nice! closed by #257 and released in [menubar@7.1.0](https://www.npmjs.com/package/menubar/v/7.1.0) 🚀
